files/initramfs-tools/union-mount.j2

```diff
@@ -26,5 +51,16 @@ mount --bind ${rootmnt}/host/$image_dir/{{ DOCKERFS_DIR }} ${rootmnt}/var/lib/do
 ## Mount the boot directory in the raw partition, bypass the overlay
 mkdir -p ${rootmnt}/boot
 mount --bind ${rootmnt}/host/$image_dir/boot ${rootmnt}/boot
-## Mount loop device for /var/log
-[ -f ${rootmnt}/host/disk-img/var-log.ext4 ] && mount -t ext4 -o loop,rw ${rootmnt}/host/disk-img/var-log.ext4 ${rootmnt}/var/log
+## Mount loop device or tmpfs for /var/log
+onie_platform=""
+aboot_platform=""
+. ${rootmnt}/host/machine.conf
+if [ X"$aboot_platform" = X"x86_64-arista_7050_qx32"  ] ||
+   [ X"$aboot_platform" = X"x86_64-arista_7050_qx32s" ]
```

Comment:
same comment as above: Can we extend this to the 7060-CX32 as well?

Reply:
I think we need more time for this change, because:

- The change I put up here is tested change.
- We didn't include 7060 in this list because we didn't observe the same issue.

That doesn't mean that we won't add 7060 to the list. It just mean that we would like to have a discussion, if we decide to proceed, then we need to test it before putting up change for review.

Are you ok with this change go in as infrastructural change first and improve later? :-)

Reply:
yes, do the merge I can contribute the change later. both platforms come from the same lineage, I have both in my network and both show the same symptoms

Reply:
Thanks @MichelMoriniaux !